x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/enrich_signal_threat_matches.test.ts

```diff
@@ -455,7 +526,11 @@ describe('enrichSignalThreatMatches', () => {
       ],
     });
     const signals = getSignalsResponseMock([signalHit, otherSignalHit]);
-    const enrichedSignals = await enrichSignalThreatMatches(signals, getMatchedThreats);
+    const enrichedSignals = await enrichSignalThreatMatches(
+      signals,
+      getMatchedThreats,
+      'threat.indicator'
```

Comment:
nit: Suggest using a constant everywhere and put it in the common/constants area we have